src/storybook-utils/.babelrc

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
only the storybook utils with the preset are compiled with babel (Because of trying to compile it with webpack create weird issues and we decide not to put effort on this right now because in the and we will move the code here outside)